ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -18,22 +20,33 @@ import {getRegistrationTasksMap, RegistrationDashboard} from 'app/pages/homepage
 import {profileApi} from 'app/services/swagger-fetch-clients';
 import colors from 'app/styles/colors';
 import {hasRegisteredAccessFetch, reactStyles, ReactWrapperBase, withUserProfile} from 'app/utils';
+import {environment} from 'environments/environment';
 import {
   BillingProjectStatus,
   Profile,
 } from 'generated/fetch';
 
 export const styles = reactStyles({
   mainHeader: {
-    color: colors.white, fontSize: 28, fontWeight: 400,
+    color: colors.primary, fontSize: 28, fontWeight: 400,
     display: 'flex', letterSpacing: 'normal'
   },
-  backgroundImage: {
-    backgroundImage: 'url("/assets/images/AoU-HP-background.jpg")',
-    backgroundRepeat: 'no-repeat', backgroundSize: 'cover', height: '100%',
+  pageWrapper: {
     marginLeft: '-1rem', marginRight: '-0.6rem', display: 'flex',
     flexDirection: 'column', justifyContent: 'space-between'
   },
+  welcomeMessage: {
+    marginLeft: '3%', display: 'flex', flexDirection: 'row'
+  },
+  welcomeMessageIconRow: {
+    display: 'flex', flexDirection: 'row', alignItems: 'flex-end', marginLeft: '1rem'
```

Comment:
it would be great to extract out display: 'flex' and flexDirection since its being in so many places